src/topology/mod.rs

```diff
@@ -45,7 +45,7 @@ type BuiltBuffer = (
 #[allow(dead_code)]
 pub struct RunningTopology {
     inputs: HashMap<String, buffers::BufferInputCloner>,
-    outputs: HashMap<String, fanout::ControlChannel>,
+    pub outputs: HashMap<String, fanout::ControlChannel>,
```

Comment:
This wound up being the change required to topology, to (externally) insert tap sinks.